tests/tracking/test_fluent.py

```diff
@@ -433,6 +437,14 @@ def test_start_run_existing_run_deleted(empty_active_run_stack):
             start_run(run_id)
 
 
+def test_start_existing_run_status(empty_active_run_stack):
+    run_id = mlflow.start_run().info.run_id
+    mlflow.end_run()
+    assert MlflowClient().get_run(run_id).info.status == RunStatus.to_string(RunStatus.FINISHED)
+    restarted_run = mlflow.start_run(run_id)
+    assert restarted_run.info.status == RunStatus.to_string(RunStatus.RUNNING)
```

Comment:
Can we test that the end time seems reasonable once we end the run again? (e.g., new end time > old end time ?)

Reply:
Done!